crates/cheatable-starknet/src/constants.rs

```diff
@@ -42,13 +48,25 @@ pub const TEST_ERC20_CONTRACT_CLASS_HASH: &str = "0x1010";
 
 #[must_use]
 pub fn build_block_context() -> BlockContext {
+    // blockifier::test_utils::create_for_account_testing
+    let vm_resource_fee_cost = Arc::new(HashMap::from([
+        (constants::N_STEPS_RESOURCE.to_string(), 1_f64),
+        (HASH_BUILTIN_NAME.to_string(), 1_f64),
+        (RANGE_CHECK_BUILTIN_NAME.to_string(), 1_f64),
+        (SIGNATURE_BUILTIN_NAME.to_string(), 1_f64),
+        (BITWISE_BUILTIN_NAME.to_string(), 1_f64),
+        (POSEIDON_BUILTIN_NAME.to_string(), 1_f64),
+        (OUTPUT_BUILTIN_NAME.to_string(), 1_f64),
+        (EC_OP_BUILTIN_NAME.to_string(), 1_f64),
```

Comment:
I think we should add an issue to investigate if the costs are fine

Reply:
#362